test/es-module/test-esm-get-format.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
```

Comment:
You need to add a flag for the test to work.
```suggestion
// Flags: --expose-internals
'use strict';
```